src/map.rs

```diff
@@ -952,11 +955,7 @@ where
 
     /// ***Panics*** if `key` is not present in the map.
     fn index(&self, key: &'a Q) -> &V {
-        if let Some(v) = self.get(key) {
-            v
-        } else {
-            panic!("IndexMap: key not found")
-        }
+        self.get(key).expect("IndexMap: key not found")
```

Comment:
Style-wise I'm not sure I agree that Option::expect is a good way for a library to report a panic, I prefer the panic as it was